src/js/components/SiteSider.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Layout, Menu } from 'antd';
+import type { MenuProps } from 'antd';
+import Icon, { PieChartOutlined, SearchOutlined, SolutionOutlined } from '@ant-design/icons';
+const { Sider } = Layout;
+
+import BeaconSvg from '@/components/Beacon/BeaconSvg';
+import { getCurrentPage, useAppSelector, useTranslationDefault } from '@/hooks';
+import { buildQueryParamsUrl } from '@/utils/search';
+
+type CustomIconComponentProps = React.ComponentProps<typeof Icon>;
+type MenuItem = Required<MenuProps>['items'][number];
+type OnClick = MenuProps['onClick'];
+
+const BeaconLogo: React.FC<Partial<CustomIconComponentProps>> = (props) => <Icon component={BeaconSvg} {...props} />;
+
+const SiteSider: React.FC = () => {
+  const navigate = useNavigate();
+  const td = useTranslationDefault();
+  const [collapsed, setCollapsed] = useState(false);
+  const queryParams = useAppSelector((state) => state.query.queryParams);
+
+  const currentPage = getCurrentPage();
+  console.log({ currentPage });
```

Comment:
make this a proper debug log with information about whats being logged or remove it